### ISSUE TITLE
RuntimeError: Failed opening file /var/infinity/log/infinity.log for writing: No such file or directory[Bug]: 

### ISSUE THREAD

**Issue by zebin-dm** [infiniflow/infinity#1607]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Version or Commit ID

0.3.0.dev3

### Other environment information

```Markdown
NAME="Ubuntu"
VERSION="20.04.6 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.6 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

### Actual behavior and How to reproduce it

import infinity

infinity_obj = infinity.connect("/home/xx/output/infinity")

### Expected behavior

Traceback (most recent call last):
  File "/home/dm/workspace/coding/python/test_infinity.py", line 3, in <module>
    infinity_obj = infinity.connect("/home/dm/workspace/coding/python/output/infinity")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dm/anaconda3/lib/python3.12/site-packages/infinity/__init__.py", line 33, in connect
    return LocalInfinityConnection(uri)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dm/anaconda3/lib/python3.12/site-packages/infinity/local_infinity/infinity.py", line 23, in __init__
    self._client = LocalInfinityClient(uri)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dm/anaconda3/lib/python3.12/site-packages/infinity/local_infinity/client.py", line 37, in __init__
    Infinity.LocalInit(path)
RuntimeError: Failed opening file /var/infinity/log/infinity.log for writing: No such file or directory
Exception ignored in: <function LocalInfinityClient.__del__ at 0x7fb9a0b34b80>
Traceback (most recent call last):
  File "/home/dm/anaconda3/lib/python3.12/site-packages/infinity/local_infinity/client.py", line 41, in __del__
    if self.client is not None:
       ^^^^^^^^^^^
AttributeError: 'LocalInfinityClient' object has no attribute 'client'
Exception ignored in: <function LocalInfinityConnection.__del__ at 0x7fb9a0b65620>
Traceback (most recent call last):
  File "/home/dm/anaconda3/lib/python3.12/site-packages/infinity/local_infinity/infinity.py", line 34, in __del__
    if self._is_connected is True:
       ^^^^^^^^^^^^^^^^^^
AttributeError: 'LocalInfinityConnection' object has no attribute '_is_connected'

**Comment by JinHai-CN** [comment]
Once the PR is merged, python SDK with 0.3.0.dev4 will be released.

**Comment by JinHai-CN** [comment]
> Once the PR is merged, python SDK with 0.3.0.dev4 will be released.

0.3.0.dev4 is released.
